Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -222,10 +222,9 @@ static void registerUserDefaults()
 
 void WebProcessPool::updateProcessSuppressionState()
 {
-    WebsiteDataStore::forEachWebsiteDataStore([enabled = processSuppressionEnabled()] (WebsiteDataStore& dataStore) {
-        if (auto* networkProcess = dataStore.networkProcessIfExists())
-            networkProcess->setProcessSuppressionEnabled(enabled);
-    });
+    bool enabled = processSuppressionEnabled();
+    for (Ref networkProcess : NetworkProcessProxy::allNetworkProcesses())
```

Comment:
Since this is Cocoa code, you could in theory just rely on `NetworkProcessProxy::defaultNetworkProcess()`.

Reply:
Yes that would work too; it just feels more consistent to use `NetworkProcessProxy::allNetworkProcesses` as other places.